build/test/TestPackageProjects.targets

```diff
@@ -71,54 +72,62 @@
     <ItemGroup>
       <BaseTestPackageProject Include="TestAssets/TestPackages/PackageWithFakeNativeDep">
         <Name>PackageWithFakeNativeDep</Name>
+        <ProjectName>PackageWithFakeNativeDep.csproj</ProjectName>
         <IsTool>False</IsTool>
         <IsApplicable>True</IsApplicable>
         <VersionPrefix>1.0.0</VersionPrefix>
-        <VersionSuffix>$(TestPackageBuildVersionSuffix)</VersionSuffix>
+        <VersionSuffix></VersionSuffix>
         <Clean>True</Clean>
         <Frameworks>net45</Frameworks>
+        <PackRuntime>$(CoreCLRRid)</PackRuntime>
       </BaseTestPackageProject>
       <BaseTestPackageProject Include="TestAssets/TestPackages/dotnet-dependency-context-test">
         <Name>dotnet-dependency-context-test</Name>
+        <ProjectName>dotnet-dependency-context-test.csproj</ProjectName>
         <IsTool>True</IsTool>
         <IsApplicable>True</IsApplicable>
         <VersionPrefix>1.0.0-rc-</VersionPrefix>
-        <VersionSuffix>$(TestPackageBuildVersionSuffix)</VersionSuffix>
+        <VersionSuffix>rc-$(TestPackageBuildVersionSuffix)</VersionSuffix>
         <Clean>True</Clean>
         <Frameworks>netcoreapp1.0</Frameworks>
       </BaseTestPackageProject>
       <BaseTestPackageProject Include="TestAssets/TestPackages/dotnet-dependency-tool-invoker">
         <Name>dotnet-dependency-tool-invoker</Name>
+        <ProjectName>dotnet-dependency-tool-invoker.csproj</ProjectName>
         <IsTool>True</IsTool>
         <IsApplicable>True</IsApplicable>
         <VersionPrefix>1.0.0-rc-</VersionPrefix>
-        <VersionSuffix>$(TestPackageBuildVersionSuffix)</VersionSuffix>
+        <VersionSuffix>rc-$(TestPackageBuildVersionSuffix)</VersionSuffix>
         <Clean>True</Clean>
         <Frameworks>netcoreapp1.0</Frameworks>
       </BaseTestPackageProject>
       <BaseTestPackageProject Include="TestAssets/TestPackages/dotnet-desktop-and-portable">
         <Name>dotnet-desktop-and-portable</Name>
+        <ProjectName>dotnet-desktop-and-portable.csproj</ProjectName>
         <IsTool>True</IsTool>
         <IsApplicable>True</IsApplicable>
         <VersionPrefix>1.0.0-rc-</VersionPrefix>
-        <VersionSuffix>$(TestPackageBuildVersionSuffix)</VersionSuffix>
+        <VersionSuffix>rc-$(TestPackageBuildVersionSuffix)</VersionSuffix>
         <Clean>True</Clean>
         <Frameworks>net451;netcoreapp1.0</Frameworks>
+        <PackRuntime>$(CoreCLRRid)</PackRuntime>
       </BaseTestPackageProject>
       <BaseTestPackageProject Include="TestAssets/TestPackages/dotnet-desktop-binding-redirects"
                           Condition=" '$(IsDesktopAvailable)' == 'True' ">
         <Name>dotnet-desktop-binding-redirects</Name>
+        <ProjectName>dotnet-desktop-binding-redirects.csproj</ProjectName>
         <IsTool>True</IsTool>
-        <IsMsbuild>True</IsMsbuild>
+		<IsMsbuild>True</IsMsbuild>
```

Comment:
In a separate one (include with my next change) so I don't have to wait for tests again before merging.